src/components/shared/pagination/DataToDisplay.tsx

```diff
@@ -0,0 +1,17 @@
+export const data = [
+  'Item 1',
+  'Item 2',
+  'Item 3',
+  'Item 4',
+  'Item 5',
+  'Item 6',
+  'Item 7',
+  'Item 8',
+  'Item 9',
+  'Item 10',
+  'Item 11',
+  'Item 12',
+  'Item 13',
+  'Item 14',
+  'Item 15',
+];
```

Comment:
If you choose to store the data like this, please move it into the `data` folder. Thanks.